### PR TITLE
RSWEB-5187 - Move fonticon hosting to cdn. [Needs Review]

### PR DESCRIPTION
**Pull request by aedile**
A few quick updates to host font icon on cdn.  Currently something wrong with the CDN links so DO NOT MERGE.

EDIT: Fixed, thanks @bighappyface !

### CODE REVIEW

File: styleguide/_sass/_rsweb-font-style.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: 'rsweb';
-    src:    url('../fonts/rsweb.eot?ezv9m0');
-    src:    url('../fonts/rsweb.eot?ezv9m0#iefix') format('embedded-opentype'),
-        url('../fonts/rsweb.ttf?ezv9m0') format('truetype'),
-        url('../fonts/rsweb.woff?ezv9m0') format('woff'),
-        url('../fonts/rsweb.svg?ezv9m0#rsweb') format('svg');
+    src:    url('https://0d0fbe434114a2d40b45-de9ed8c45372075d79906316dada8cf3.ssl.cf1.rackcdn.com/icon/rsweb.eot?ezv9m0');
```
Comment (bighappyface) [inline styleguide/_sass/_rsweb-font-style.scss]: The protocol should be removable here and what's up with the `ezv9m0` query string?

Comment (aedile) [inline styleguide/_sass/_rsweb-font-style.scss]: No clue on the querystring.  I preserved from previous version.  I'll ask Amber.

Comment (aedile) [inline styleguide/_sass/_rsweb-font-style.scss]: Amber didn't have any perspective on the qstrings.  Gutcheck says it was to bust cache.  I removed them and they work just fine (although the #iefix hashtag should stay).  

Re: the protocol, the files are hosted directly from the SSL version of the container, so mixed protocol won't work here.  Especially when you are testing local, which is not https.  Confirmed via visual test that stripping the protocol breaks this.

**Comment by jhthompson** [comment]
+1 Looks like its working now!

File: styleguide/_sass/_rsweb-font-style.scss (hunk 1 above)
Comment (bighappyface) [inline styleguide/_sass/_rsweb-font-style.scss]: Good call on the protocol being hostname-specific. Gotta slow down on reviews sometimes

**Comment by xinbin** [comment]
This has been tested. I can't merge this though, how do I find a person with write access? eh eh :)

**Pull request merged**